cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/KafkaRoller.java

```diff
@@ -568,14 +568,15 @@ protected Config brokerLogging(int brokerId) throws ForceableProblem, Interrupte
         );
     }
 
-    protected void dynamicUpdateBrokerConfig(int podId, Admin ac, KafkaBrokerConfigurationDiff configurationDiff, KafkaBrokerLoggingConfigurationDiff logDiff)
+    protected void dynamicUpdateBrokerConfig(PodRef podRef, Admin ac, KafkaBrokerConfigurationDiff configurationDiff, KafkaBrokerLoggingConfigurationDiff logDiff)
             throws ForceableProblem, InterruptedException {
         Map<ConfigResource, Collection<AlterConfigOp>> updatedConfig = new HashMap<>(2);
+        var podId = podRef.podId;
         updatedConfig.put(Util.getBrokersConfig(podId), configurationDiff.getConfigDiff());
         updatedConfig.put(Util.getBrokersLogging(podId), logDiff.getLoggingDiff());
 
-        LOGGER.debugCr(reconciliation, "Altering broker configuration {}", podId);
-        LOGGER.traceCr(reconciliation, "Altering broker configuration {} with {}", podId, updatedConfig);
+        LOGGER.debugCr(reconciliation, "Altering broker configuration {}", podRef);
```

Comment:
Elsewhere in this PR we use "dynamic update", but here the verb is suddenly "altering". Wouldn't "updating" be more consistent?

Also personally I think the "dynamic" isn't really necessary.

Reply:
done